packages/react/src/evm/components/MediaRenderer.tsx

```diff
@@ -445,6 +457,30 @@ export const MediaRenderer = React.forwardRef<
           {...restProps}
         />
       );
+    } else if (videoOrImageSrc.mimeType.startsWith("model")) {
+      return (
+        <Suspense
+          fallback={
+            poster ? (
```

Comment:
I don't think this is necessary, we can pass the poster to the ModelViewer, <model-viewer> accepts a prop poster that does exactly what we want